samples/java-customer-registry/src/main/java/customer/CustomerValueEntityFactory.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package customer;
+
+import com.akkaserverless.javasdk.valueentity.ValueEntityContext;
+import com.akkaserverless.javasdk.valueentity.ValueEntityFactory;
+import com.akkaserverless.javasdk.valueentity.ValueEntityOptions;
+import com.google.protobuf.Descriptors;
+import customer.api.CustomerApi;
+
+public class CustomerValueEntityFactory implements ValueEntityFactory<CustomerValueEntity> {
```

Comment:
This factory brings in all the pieces needed to register and instantiate the entity.

This is generated code, but lives in `src/main/java`. Users can add a constructor and pass in whatever they prefer.